code/workspaces/web-app/src/containers/adminMessages/AdminMessagesContainer.js

```diff
@@ -36,7 +68,13 @@ const AdminMessagesContainer = () => {
 
   return (
     <div>
-      {messages.map(m => <AdminMessage message={m} key={m.id} deleteMessage={confirmDeleteMessage(dispatch)}/>)}
+      <MuiPickersUtilsProvider utils={MomentUtils}>
```

Comment:
My only question is whether this can be hoisted to `app.js` to act as a provider for the whole app.